src/science/landuse/landuse3.F90

```diff
@@ -3,8 +3,49 @@ MODULE landuse_variable
   IMPLICIT NONE
 
   SAVE
-
+  !*# Overview of `landuse_variable` module (landuse3.F90)
+  !
+  ! This MODULE applies land-use changes for CABLE.
+  !
+  ! This includes:
+  !
+  ! - plant functional type transitions
+  ! - wood harvest
+  ! - land management
+  !
+  !## Method
+  !
+  ! All variables in the restart file and the state variables will be
+  ! calculated.
+  !
+  ! There are three groups of `landuse_mland` member variables. For example,
+  ! let's say we have a variable named `var`, then:
+  !
+  ! - `luc%var_x(mland,mvmax)` contains data before land-use change,
+  !    indexed using `mland` and `mvmax`.
+  ! - `luc%var_y(mland,mvmax)` contains data after land-use change, indexed
+  !    using `mland` and `mvmax`.
+  ! - `lucmp%var(mp)` contains the variable data, indexed using the patch
+  !    number from 1 to `mp`.
```

Comment:
```suggestion
  ! - `luc%var_x(mland,mvmax)` contains data before land-use change,
  !    indexed on the number of land points (`mland`) and the maximum number of plant functional types (`mvmax`).
  ! - `luc%var_y(mland,mvmax)` contains data after land-use change, indexed
  !    on the number of land points (`mland`) and the maximum number of plant functional types (`mvmax`).
  ! - `lucmp%var(mp)` contains the variable data, indexed on the number of vegetation patches (`mp`).
```

Reply:
The reason I split the lines was to keep lines less than 80 characters long. 80 character lines is the style specified in the JULES/UM style guidelines (I think...). However, I've always thought this restriction is silly. Hardly anyone owns a monitor anymore that is restricted to 80 characters. Unrestricted line length is annoying though. I usually find that 100 character lines is enough to provide ample space for code and comments, which can be displayed comfortably on a modern display, with a vertically split window at 11 point font. What do you think?